src/scss/pages/_login.scss

```diff
@@ -80,3 +87,29 @@
     margin-bottom: 10px;
   }
 }
+.googleIcon{
+  height: 25px;
+  float: left;
+}
+
+@media (max-width: $breakpointMobile){
```

Comment:
I'm getting a `SassError: Undefined variable: "$breakpointMobile".` when trying to run this locally 🤔 
We should add this variable to /scss/variables.scss and define it according to the common breakpoints. There might also not be a catch-all breakpoint here either, so we might need to adjust after playing with the sizes

Reply:
Interesting! I wasn't getting that error on my machine. I'll look into it.

I defined $breakpointMobile as 1200px, and had success in the past using this as the general 'mobile screen size'. There could easily be a better way!

Reply:
@Chris-Boe that was my bad, I had originally defined $breakpointMobile within scss/variables.scss but forgot to commit. I just pushed up a couple more commits containing that global styling variable along with some minor changes to the 'Login With Google" button.  

I did a bit of research online before defining $breakpointMobile and 1200px ends up being a good rule of thumb. Should we need to change it in the future we always can :)